databases/week3/mealsharing.sql

```diff
@@ -0,0 +1,179 @@
+
+-- Delete a meal with any id, fx 1
+DELETE FROM meal WHERE id = 2;
+
+
+CREATE TABLE `reservation` (
+ `id` int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+ `number_of_guests` int(10) NOT NULL,
+ `meal_id` int(10) unsigned NOT NULL,
+ `location` varchar(255) NOT NULL,
+ `created_date` datetime NOT NULL,
+ CONSTRAINT `fk_reservation_meal_id` FOREIGN KEY (`meal_id`) REFERENCES `meal` (`id`) ON DELETE CASCADE ON UPDATE CASCADE
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+INSERT INTO reservation 
+VALUES (1, 4, 1, 'WA', '2020-06-01'),
+	(2, 6, 2, 'CA', '2020-05-31'),
+    (3, 2, 3, 'CO', '2020-05-30');
+    
+INSERT INTO reservation 
+VALUES (5, 4, 5, 'CO', '2020-05-28'),
+	(6, 6, 6, 'LA', '2020-05-22'),
+    (7, 2, 7, 'UT', '2020-05-26'),
+    (8, 4, 8, 'PX', '2020-05-23'),
+	(9, 6, 9, 'MI', '2020-05-15'),
+    (10, 2, 10, 'NJ', '2020-05-17');
+    
+-- Reservation queries
+
+-- Get all reservations
+SELECT * FROM reservation;
+
+-- Add a new reservation
+INSERT INTO reservation
+VALUES (4, 4, 4, 'PX', '2020-05-25');
+
+-- Get a reservation with any id, fx 1
+SELECT * FROM reservation WHERE id = 1;
+
+-- Update a reservation with any id, 
+-- fx 1. Update any attribute fx the title or multiple attributes
+UPDATE reservation SET meal_id = 1 WHERE meal_id = 4;
+
+-- Delete a reservation with any id, fx 1
+DELETE FROM reservation WHERE id = 1;
+
+CREATE TABLE `review` (
+ `id` int(10) unsigned NOT NULL AUTO_INCREMENT PRIMARY KEY,
+ `title`  varchar(255) NOT NULL,
+ `description` text NOT NULL,
+ `meal_id` int(10) unsigned NOT NULL,
+ `stars` int(10) NOT NULL,
+ `created_date` datetime NOT NULL,
+ CONSTRAINT `fk_review_meal_id` FOREIGN KEY (`meal_id`) REFERENCES `meal` (`id`) ON DELETE CASCADE ON UPDATE CASCADE
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci;
+
+INSERT INTO review 
+VALUES (1, 'Ordered Lasagna', 'tasty and and enough for 4 ppl', 1, 5, '2020-06-01'),
+	(2, 'We had Rice poriddge', 'perfect for a small party as a dessert', 2, 4, '2020-05-31'),
+    (3, 'Wok noodle was tasty', 'good for busy week night dinner', 3, 4, '2020-05-30');
+    
+INSERT INTO review 
+VALUES (5, 'Waffel', 'tasty and and enough for 4 ppl', 5, 3, '2020-06-01'),
+	(6, 'Pizza', 'ordered for a party and big enough', 6, 4, '2020-05-31'),
+    (7, 'Curry', 'delicious japanese curry', 7, 4, '2020-05-30'),
+    (8, 'Sushi', 'healthy and enviromental friendly', 8, 5, '2020-06-01'),
+	(9, 'Cake', 'not too sweet', 9, 4, '2020-05-31'),
+    (10, 'Bread', 'tasty home made bread', 10, 5, '2020-05-30');
+
+-- Review queries
+
+-- Get all reviews
+SELECT * FROM review;
+
+-- Add a new review
+INSERT INTO review 
+VALUES (4, 'Great wok so far', 'tasty and cheap', 3, 4, '2020-05-31');
+
+-- Get a review with any id, fx 1
+SELECT * FROM review WHERE id = 4;
+
+-- Update a review with any id, 
+-- fx 1. Update any attribute fx the title or multiple attributes
+UPDATE review SET starts = 3 WHERE id = 4;
```

Comment:
You have a small typo in this one. It gives me this error: `Error Code: 1054. Unknown column 'starts' in 'field list'`